composer/datasets/synthetic.py

```diff
@@ -168,7 +169,7 @@ class SyntheticDatasetHparams(DatasetHparams):
     def validate(self):
         super().validate()
 
-        _validate_label_inputs(label_type=self.label_type, num_classes=self.num_classes, label_shape=self.label_shape)
+        validate_label_inputs(label_type=self.label_type, num_classes=self.num_classes, label_shape=self.label_shape)
```

Comment:
If validation happens in `__init__`, does it need to happen again in `validate()`?